src/com/google/javascript/jscomp/ProcessCommonJSModules.java

```diff
@@ -666,7 +666,7 @@ private void visitExport(NodeTraversal t, Node export) {
       }
 
       if (root.getParent().isAssign()
-          && (root.getNext().isName() || root.getNext().isGetProp())
+          && (root.getNext() != null && (root.getNext().isName() || root.getNext().isGetProp()))
```

Comment:
It failed on this line https://github.com/shama/bel/blob/master/index.js#L1 but I had no idea why.

You will need `splittable` to compile the package for now. I implemented a subset of the `browser` key in `package.json` as an overlay. Is this something you had on your radar?